examples/tutorials/Gpaw_cell_tutorial.ipynb

```diff
@@ -0,0 +1,343 @@
+{
```

Comment:
<p>Add a picture of the silicon unit cell. Add a reference and add some discussion</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/4219/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>